theseus/embodied/motionmodel/double_integrator.py

```diff
@@ -28,6 +28,12 @@ def __init__(
             raise ValueError(
                 "All variables for a DoubleIntegrator must have the same dimension."
             )
+        if not isinstance(dt, Variable):
+            if not isinstance(dt, torch.Tensor):
+                dt = torch.tensor(dt)
+            self.dt = Variable(dt)
+        else:
+            self.dt = dt
```

Comment:
After the `ndim` check below, we can add `dt.data = self.dt.data.view(-1, 1)`, to make sure we have a batch dimension as well, as discussed above.